.github/workflows/build-streams.yaml

```diff
@@ -7,25 +7,122 @@ on:
       - release-sentry-streams/**
 
 jobs:
-  dist:
-    name: Create wheel and source distribution for sentry-streams
-    timeout-minutes: 10
+  linux:
+    permissions: read-all
+    timeout-minutes: 5
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: ubuntu-latest
+            target: x86_64
+          - runner: ubuntu-latest-arm
+            target: aarch64
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+      - name: Build wheels
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.platform.target }}
+          args: --release --out dist --find-interpreter
+          sccache: ${{ !startsWith(github.ref, 'refs/tags/') }}
+          manylinux: auto
+          working-directory: ./sentry_streams
+          before-script-linux: |
+            cmake --version
+            # If we're running on rhel centos, install needed packages.
+            if command -v yum &> /dev/null; then
+                yum update -y && yum install -y openssl openssl-devel pkgconfig
+
+            else
+                # If we're running on debian-based system.
+                apt update -y && apt-get install -y libssl-dev openssl pkg-config
+            fi
+          # Settings MAKE_POLICY_VERSION_MINIMUM to 3.5 the librdkafka imported
+          # by rdkafka does not compile with cmake 4 provided by the maturin action.
+          docker-options:
+            -e CMAKE_POLICY_VERSION_MINIMUM=3.5
+      - name: Upload wheels
+        uses: actions/upload-artifact@v4
+        with:
+          name: wheels-linux-${{ matrix.platform.target }}
+          path: ./sentry_streams/dist
+
+
+  macos:
+    permissions: read-all
+    timeout-minutes: 5
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: macos-13
```

Comment:
Is there any reason why we're using macos-13 for x86 and macos-latest for arm?

Reply:
no, fixed.